GoogleCastOptions type is not exported

### Current Behavior:

`GoogleCastOptions` is used as the type for the `googleCast` property on the public `MediaPlayerProps` interface, but it is not exported from the `vidstack` package. Since TypeScript cannot "name" the type in declaration output, it throws TS4023:

```
error TS4023: Exported variable '__VLS_export' has or is using name
'GoogleCastOptions' from external module ".../vidstack/types/vidstack-tX8MEPiY" but cannot be named.
```

### Expected Behavior:

`GoogleCastOptions` should be exported from the `vidstack` package alongside the other public types since it is referenced in the public `MediaPlayerProps` interface. No errors should occur when `vidstack/vue` types are imported.

### Steps To Reproduce:

1. Create a blank Vue + TypeScript project with the following dependencies:
   ```json
   {
     "dependencies": {
       "vidstack": "1.12.13",
       "vue": "3.5.30"
     },
     "devDependencies": {
       "typescript": "5.9.3",
       "vue-tsc": "3.2.5"
     }
   }
   ```
2. Use this `tsconfig.json` (both `composite: true` and `vueCompilerOptions` must be present):
   ```json
   {
     "compilerOptions": {
       "composite": true,
       "types": ["vidstack/vue"]
     },
     "vueCompilerOptions": {
       "strictTemplates": true,
       "fallthroughAttributes": true
     }
   }
   ```
3. Create `src/App.vue`:
   ```vue
   <template>
     <media-player :title="title" :src="src">
       <media-provider />
     </media-player>
   </template>

   <script lang="ts" setup>
   withDefaults(
     defineProps<{
       title?: string;
       src?: string;
     }>(),
     {
       title: undefined,
       src: undefined,
     }
   );
   </script>
   ```
4. Run `npx vue-tsc --noEmit`
5. See TS4023 error referencing `GoogleCastOptions`

**Reproduction Link:** [How to create a repro?][repro]

[repro]: https://github.com/vidstack/player/blob/main/.github/REPRO.md

### Environment:

- Framework: Vue 3.5.30
- vue-tsc: 3.2.5
- TypeScript: 5.9.3
- vidstack: 1.12.13
- Node: 24

### Anything Else?

`GoogleCastOptions` is defined in `src/providers/google-cast/types.ts` and imported in `src/core/api/player-props.ts`, but the Google Cast section in `src/exports/providers.ts` only exports the loader, provider, and events — not the types. HLS and DASH both export their types files (`export type * from '.../<provider>/types'`), so the fix is to add the same line for Google Cast.

--- a/packages/vidstack/src/elements/define/chapter-title-element.ts
+++ b/packages/vidstack/src/elements/define/chapter-title-element.ts
@@ -4,7 +4,7 @@ import { Host } from 'maverick.js/element';
 import { useMediaContext, type MediaContext } from '../../core/api/media-context';
 import { watchCueTextChange } from '../../core/tracks/text/utils';
 
-interface ChapterTitleProps {
+export interface ChapterTitleProps {
   /**
    * Specify text to be displayed when no chapter title is available.
    */

--- a/packages/vidstack/src/exports/components.ts
+++ b/packages/vidstack/src/exports/components.ts
@@ -77,3 +77,8 @@ export type {
   PlyrLayoutTranslations,
 } from '../components/layouts/plyr/translations';
 export { usePlyrLayoutClasses } from '../components/layouts/plyr/plyr-layout';
+
+// Elements
+export type { ChapterTitleProps } from '../elements/define/chapter-title-element';
+export type { MediaLayoutProps } from '../elements/define/layouts/layout-element';
+export type { SpinnerProps } from '../elements/define/spinner-element';

--- a/packages/vidstack/src/exports/providers.ts
+++ b/packages/vidstack/src/exports/providers.ts
@@ -32,6 +32,7 @@ export type { DASHProvider } from '../providers/dash/provider';
 export type { GoogleCastLoader } from '../providers/google-cast/loader';
 export type { GoogleCastProvider } from '../providers/google-cast/provider';
 export type * from '../providers/google-cast/events';
+export type * from '../providers/google-cast/types';
 
 // Vimeo
 export { VimeoProviderLoader } from '../providers/vimeo/loader';

